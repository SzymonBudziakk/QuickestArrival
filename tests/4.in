100 100
*##################################################################################################*
#AAA......#.........#.........#.........#.........#.........#.........#.........#.........#.....DDD#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#EEE......#.........#.........#.........#.........#.........#........#
##############################*#####################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
####################################################################################################
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#....FFF..#........#
#####################################################################################*##############
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#.........#.........#.........#.........#.........#.........#.........#.........#.........#........#
#CCC......#.........#.........#.........#.........#.........#.........#.........#.........#.....BBB#
*##################################################################################################*
0
36
AAA AAA 0
AAA BBB 0
AAA CCC 0
AAA DDD 0
AAA EEE 0
AAA FFF 0
BBB AAA 0
BBB BBB 0
BBB CCC 0
BBB DDD 0
BBB EEE 0
BBB FFF 0
CCC AAA 0
CCC BBB 0
CCC CCC 0
CCC DDD 0
CCC EEE 0
CCC FFF 0
DDD AAA 0
DDD BBB 0
DDD CCC 0
DDD DDD 0
DDD EEE 0
DDD FFF 0
EEE AAA 0
EEE BBB 0
EEE CCC 0
EEE DDD 0
EEE EEE 0
EEE FFF 0
FFF AAA 0
FFF BBB 0
FFF CCC 0
FFF DDD 0
FFF EEE 0
FFF FFF 0
