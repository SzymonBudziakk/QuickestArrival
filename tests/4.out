0
198
99
99
89
174
198
0
99
99
109
24
99
99
0
198
70
95
99
99
198
0
128
103
89
109
70
128
0
85
174
24
95
103
85
0
